### PR TITLE
Update e107_admin/plugin.php

### PR DESCRIPTION
**Pull request by yesszus**
Add missing lang codes.

### CODE REVIEW

File: e107_languages/English/admin/lan_plugin.php [hunk 1]
```diff
@@ -306,7 +306,7 @@
 define("LAN_INSTALL_FAIL", "Installation failed!");
 define("LAN_UNINSTALL_FAIL", "Unable to uninstall!");
 define("LAN_PLUGIN_IS_USED", "[x] plugin is used by:");
-
+define("LAN_PATH", "Path");
```
Comment (CaMer0n) [inline e107_languages/English/admin/lan_plugin.php]: It should be in the non-universal format: EPL_ADLAN_XX 
The ONLY files which should contain the non-numeric format should be lan_admin.php and e107_languages/English/English.php . Please keep this in mind when adding new LANs. Thank you!

Comment (yesszus) [inline e107_languages/English/admin/lan_plugin.php]: Ok! Moved to lan_admin.php.

File: e107_languages/English/admin/lan_admin.php [hunk 2]
```diff
@@ -328,7 +328,7 @@
 define("LAN_VIEW_ALL","View All");
 
 define("LAN_BAN","Ban");
-
+define("LAN_PATH", "Path");
```
Comment (CaMer0n) [inline e107_languages/English/admin/lan_admin.php]: Unless 'path' is used often throughout the admin of e107, (I don't think it is) this should be in lan_plugin.php 
What do you think @MikeyGMT ?

Comment (MikeyGMT) [inline e107_languages/English/admin/lan_admin.php]: It's defined more than once:- 
define("FMLAN_29", "Path");
define("IMALAN_110", "Path"); 

There's a few Path to [x] candidates too.
There may be some synonyms, e.g. directory, folder, location, url, which could be reviewed to use a consistent term.

Comment (CaMer0n) [inline e107_languages/English/admin/lan_admin.php]: okay!! Thank you!! 👍

**Pull request merged**